aptos-move/framework/aptos-framework/sources/transaction_context.spec.move

```diff
@@ -11,9 +14,26 @@ spec aptos_framework::transaction_context {
         ensures result == spec_get_txn_hash();
     }
     spec fun spec_get_txn_hash(): vector<u8>;
+    spec get_transaction_hash(): vector<u8> {
+        pragma opaque;
+        // property 1: Fetching the transaction hash should return a vector with 32 bytes, if the auid feature flag is enabled.
+        aborts_if [abstract] !std::features::auids_enabled();
```

Comment:
We don't need this now, so you remove it. Please check the updated doc.

Reply:
sure